contract_compassion/model/product.py

```diff
@@ -0,0 +1,36 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+from openerp.osv import orm, fields
+from openerp.tools.translate import _
+import pdb
+
+class product(orm.Model):
+    _inherit = 'product.product'
+
+    _columns = {
+        'type': fields.selection([

```

Comment:
Is it needed to add a product type ? If yes, you could add the type 'G', 'Gift', which would be useful everywhere I check product.name is in gift names...